CatalogReaderExample/NuGet.Protocol.Catalog.Sample/Program.cs

```diff
@@ -20,8 +20,9 @@ private static async Task MainAsync()
             using (var loggerFactory = new LoggerFactory().AddConsole(LogLevel.Warning))
             using (var httpClient = new HttpClient())
             {
+                var jsonClient = new SimpleHttpClient(httpClient, loggerFactory.CreateLogger<SimpleHttpClient>());
                 var fileCursor = new FileCursor("cursor.json", loggerFactory.CreateLogger<FileCursor>());
-                var catalogClient = new CatalogClient(httpClient, loggerFactory.CreateLogger<CatalogClient>());
+                var catalogClient = new CatalogClient(jsonClient, loggerFactory.CreateLogger<CatalogClient>());
```

Comment:
Naming is hard. I stole `jsonClient` from here: https://github.com/NuGet/NuGet.Jobs/blob/9d2ed964202c466110c5c9f18e48bcb76f9ede17/src/NuGet.Protocol.Catalog/CatalogClient.cs#L15

I was seriously consider `httpClient2`. Please let me know if you have better suggestions :P

Reply:
`simpleHttpClient`?

Reply:
Done!